### ISSUE TITLE
Rethink "Seen on [date]" in list of seen movies

### ISSUE THREAD

**Issue by wtimme** [spacepandas/cineaste-ios#105]
**Is your feature request related to a problem? Please describe.**
When I start using the app, I want to put movies that I have already seen on the (empty) "History" list. After adding a movie, the list says "Seen on [current date]". This is not correct - I have only _marked_ it as seen at that date.

![seen](https://user-images.githubusercontent.com/1681085/61188348-3148a000-a66d-11e9-96cf-596cab37c4d5.jpeg)

(I've seen this movie a couple of years back - not just some minutes ago.)

**Describe the solution you'd like**
For the MVP, I propose we rephrase this to "Added on [current date]".

**Describe alternatives you've considered**
I might not be getting the purpose of this "History" screen.

**Additional context**
Personally, I initially thought of this as a list of movies that I watched and can recommend. Especially since the "call to action" in the empty state is worded like this:

> You don't have any movies you have already seen on this list.
> Go ahead and add a new movie.

As a new user, this sounds like I could have a collection of movie recommendations.

**Comment by fbernutz** [comment]
I really like the idea of just changing the wording. There is another issue open #90 where you could change the date on which you watched the movie. But until this feature is implemented, I love the wording suggestion! Do you want to make a PR for this?
